### ISSUE TITLE
revert default index.max_result_window back to 10000

### ISSUE THREAD

**Issue by rymsha** [enonic/xp#7951]
we must revert https://github.com/enonic/xp/commit/00287ab169d2b930fa1b44a6a9c2e7c5fc484834 (partially) 
index.max_result_window was increased to 500000 as a quick-fix for issue #7124

but after #7934 it is no longer needed.

Problems with this quick-fix are:
- it is a possible reason for #7290
- It is quite easy to get OOME, by requesting, for instance from= 499900 to = 500000, but there is no sufficient heap (I assume it is a DoS security issue as well).
- it is still limited to 500000 😆 

But there is a problem with revert as well. Some customers may get used to almost "infinite" pagination experience out of the box. But for such cases they can override default value via configuration. And all already created indices won't read this value either (customers can change this value directly on each index, if absolutely required)

In XP 8 this is will most likely won't be relevant (this setting will is gone in ES7)

Also, read
https://discuss.elastic.co/t/index-max-result-window-not-working/130509
